main/FatesParametersInterface.F90

```diff
@@ -53,19 +54,21 @@ module FatesParametersInterface
      type(parameter_type), private :: parameters(max_params)
 
    contains
```

Comment:
@glemieux , in the following block of changes, I was surprised that previously listed procedures that were explicitly public were switched to undefined (no label).  Were these not used globally?  Also, private functions are listed down below, which are implicitly private now, right?

Reply:
ok, just saw that those were bound inside a type, which is declared as public, (@glemieux showed me)